Cargo.toml

```diff
@@ -28,6 +28,12 @@ git = 'https://github.com/paritytech/substrate.git'
 package = 'sr-io'
 rev = '0e3001a1ad6fa3d1ba7da7342a8d0d3b3facb2f3'
 
+[dependencies.old-forum]
+default_features = false
+git = 'https://github.com/juniuszhou/substrate-forum-module'
```

Comment:
For data migration, I need access each data member in category, thread and post. but they are defined as private in old forum module. I must update them in old forum module, but I don't where (we just master and development branch) to commit in joystream repo. So I temporarily use code base in my repo to make build.

Reply:
Ok, great, I think this means we must basically always must have all fields public in any module, because we will always want to do migration in the future.

Can you add this concept to the Substrate convention issue?

Reply:
https://github.com/Joystream/joystream/issues/36#issuecomment-583729262 added for the coding convention.